HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Replace_text_in_list_paragraph
```

Comment:
Change sample name as Replace text in list  paragraph with HTML

Reply:
modified